src/Uno.UWP/Devices/Geolocation/Geolocator.Android.cs

```diff
@@ -77,6 +78,21 @@ public uint ReportInterval
 		}
 	}
 
+	// This override prevents the following ObjectDisposedException crash from happening:
```

Comment:
I think you should move the stacktrace into an issue, and reference that instead.